doc/releases/changelog-0.41.0.md

```diff
@@ -4,32 +4,33 @@
 
 <h3>New features since last release</h3>
 
-* Added additional Hadamard gradient modes and `"reversed"`, `"direct"`, and `"reversed-direct"` modes are now available for use with the hadamard gradient.
-  [(#7046)](https://github.com/PennyLaneAI/pennylane/pull/7046)
-
-<h4>Resource-efficient decompositions 🔎</h4>
+<h4>Resource-efficient Decompositions 🔎</h4>
 
-A new, experimental graph-based decomposition system is now available in PennyLane under the `qml.decomposition` 
-module. 
+A new, experimental graph-based decomposition system is now available in PennyLane, offering more resource-efficiency and versatility.
```

Comment:
FYI: these PR links are all in one place because the implementation for features is kind of scattered all over the place 😅. I can dig into the PRs if needed and try my best to allocate them to specific bullet points, but didn't find this to be a useful exercise 🏋️